content/en/docs/writing-policies/cleanup.md

```diff
@@ -96,3 +136,35 @@ spec:
 Although labeled resources are watched by Kyverno, the cleanup interval (the time resolution at which any cleanup can be performed) is controlled by a flag passed to the cleanup controller called `ttlReconciliationInterval`. This value is set to `1m` by default and can be changed if a longer resolution is required.
 
 Because this is a label, there is opportunity to chain other Kyverno functionality around it. For example, it is possible to use a Kyverno mutate rule to assign this label to matching resources. A validate rule could be written prohibiting, for example, users from the `infra-ops` group from assigning the label to resources in certain Namespaces. Or, Kyverno could generate a new resource with this label as part of the resource definition.
+
+### Deletion Propagation Policy in TTL-based Cleanup
+
+The deletionPropagationPolicy can also be specified for resources with a TTL-based cleanup label. This field provides flexibility in handling dependent resources when a resource reaches its expiration. For example:
+
+- **Foreground**: Ensures that all dependent resources are deleted before the resource itself is removed.
+- **Background**: Deletes the resource first, while dependents are removed asynchronously.
+- **Orphan**: Deletes the resource but leaves its dependents in place.
+
+For example, consider a Pod with the TTL label `cleanup.kyverno.io/ttl: 2m`. After two minutes, the Pod will be deleted, but the dependents will be handled according to the specified deletionPropagationPolicy. If the policy is set to Orphan, the Pod will be deleted, but its dependent resources will remain in the cluster.
+
+```yaml
+# TTL-based cleanup with deletionPropagationPolicy
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    cleanup.kyverno.io/ttl: 2m
+  name: foo
+spec:
+  containers:
+  - args:
+    - sleep
+    - 1d
+    image: busybox:1.35
+    name: foo
+deletionPropagationPolicy: "Orphan"
+```
```

Comment:
Deletion propagation policy cannot be added to pod manifest, nor does it work with ttl label